.gitignore

```diff
@@ -8,20 +8,18 @@ __pycache__/
 coverage.xml
 
 # setuptools
-/build/**/*.*
-/dist/**/*.*
+/build/
```

Comment:
This will ignore `requirements.txt` files in `build` and `dist`.
As I tried to explain in one of our discussions, anti-pattern won't easily work if you're not using `**/*.*`.

E.g. `!requirements.txt` won't work anymore to allow such files in `build` and `dist`.

Reply:
Done.